test/Faker/Provider/PersonTest.php

```diff
@@ -2,9 +2,11 @@
 
 namespace Faker\Test\Provider;
 
-use Faker\Provider\Person;
 use Faker\Generator;
+use Faker\Provider\Person;
 use PHPUnit\Framework\TestCase;
+use ReflectionException;
+use ReflectionProperty;
```

Comment:
Personally, I'd prefer not to import classes from the root namespace, i.e, from the Standard PHP Library (see https://www.php.net/manual/en/book.spl.php).

Perhaps we can reference them relative to the root namespace instead?